easier_net/easier_net.py

```diff
@@ -0,0 +1,186 @@
+from ensurepip import bootstrap
+# from imblearn import RandomUnderSampler
+import numpy as np
+
+import pickle
+import sier_net #technically should be from sier_net import SierNetEstimator
+
+class EasierNetEstimator():
+    """ This is the Ensembling by Averaging Sparse-Input Hierarchical Networks(EASIER-NET)"""
+    #i think my init needs a seed
+    def __init__(
+        self, #num inits >=5
+        # seed=12,
+        # data_file="_output/data.npz",
+        num_classes=0,
```

Comment:
at some point, we should make an easier net for regression and one for classification. this is fine for now though